engines/wintermute/video/video_theora_player.cpp

```diff
@@ -30,7 +30,7 @@
 #include "engines/wintermute/video/video_theora_player.h"
 #include "engines/wintermute/base/base_game.h"
 #include "engines/wintermute/base/base_file_manager.h"
-#include "engines/wintermute/base/gfx/osystem/base_surface_osystem.h"
+#include "engines/wintermute/base/gfx/base_surface.h"
```

Comment:
add ENABLE_WME3D

Reply:
Done

Reply:
Disagree, instead drop both macro guards in this file, so that the only call is to createSurface.

That's how it should have worked anyhow, and 100% equivalent for the non-3D code.

Downside being that it does introduce a minor merge conflict, so I'd suggest creating a separate PR against ScummVM to get this fixed (along with the hasFile-fix)